app/__init__.py

```diff
@@ -0,0 +1,10 @@
+# Author: Micah Weiberg
+# Version: 04-15-19
+# Class that imports Flask to run the app
+from flask import Flask
+from app.views import TestView
+
+app = Flask(__name__)
+# app.config.from_object('config')
+
+TestView.register(app, route_base='/')
```

Comment:
line 10 - `TestView` should be renamed to something more meaningful, like `ChessView` or `HomeView`, etc.